sdk/keyvault/test-resources-post.ps1

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# IMPORTANT: Do not invoke this file directly. Please instead run eng/New-TestResources.ps1 from the repository root.
+
```

Comment:
I don't pretend to understand this scipt in its entirety but it enabled Managed HSM for us. Sourced from https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/test-resources-post.ps1